### PR TITLE
ADD Omemo default support

### PR DESCRIPTION
**Pull request by orbitz**

### CODE REVIEW

File: src/headless/converse-core.js [hunk 1]
```diff
@@ -223,6 +223,7 @@ _converse.default_settings = {
     ],
     message_carbons: true,
     nickname: undefined,
+    omemo_default: false,
```
Comment (jcbrand) [inline src/headless/converse-core.js]: This needs to go into [src/converse-omemo.js](https://github.com/conversejs/converse.js/blob/master/src/converse-omemo.js) and you need to use [_converse.api.settings.update](https://conversejs.org/docs/html/api/-_converse.api.settings.html#.update) inside the plugin's [initialize](https://github.com/conversejs/converse.js/blob/master/src/converse-omemo.js#L240) method.

Other than that, looks good :+1:

Comment (orbitz) [reply src/headless/converse-core.js]: How's that?

**Comment by jcbrand** [comment]
Thanks, one more thing please.

Mention the ticket number for https://github.com/conversejs/converse.js/issues/1476 in the changelog and make sure that your entry is in the right order.

Bonus points if you mention the ticket number in the commit message.

**Comment by orbitz** [comment]
Done!

**Comment by orbitz** [comment]
Looks like I broke something, I'll have to look tomorrow.

**Comment by orbitz** [comment]
I'm also not sure this is working with the latest change, does the variable go to a different place than I'm checking with the method you said to use?

**Comment by jcbrand** [comment]
Thanks!

**Pull request merged**